docs/toc.yml

```
@@ -704,23 +701,20 @@
     - name: Runtime
       href: excel/custom-functions-runtime.md
       displayName: Excel, Custom Functions
-    - name: Save and share state
-      href: excel/custom-functions-save-state.md
-      displayName: Excel, Custom Functions
     - name: Troubleshooting
       href: excel/custom-functions-troubleshooting.md
       displayName: Excel, Custom Functions
-    - name: Use a shared runtime
-      items:
-      - name: Overview of using a shared runtime
-        href: excel/custom-functions-shared-overview.md
-        displayName: Excel, Custom Functions
-      - name: Shared custom function data with the task pane tutorial
-        href: tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md
-        displayName: Excel, Custom Functions
-      - name: Call Excel JS APIs from your custom function
-        href: excel/call-excel-apis-from-custom-function.md
-        displayName: Excel, Custom Functions
+    - name: UI-less custom functions
```

```suggestion
    - name: Non-task pane (UI-less) custom functions
```
Think we only need to do this once in the TOC.